### PR TITLE
feat(CommunityPermissions): Integrate welcome community popup into `JoinCommunityView`

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes https://github.com/status-im/status-desktop/issues/9268

### What does the PR do

- It improves `CommunityIntroDialog` resizing behaviour.
- It improves storybook `CommunityIntroDialogPage`.
- It integrates `CommunityIntroDialog` into `JoinCommunityView`. 

### Affected areas

- `JoinCommunityView` and `CommunityIntroDialog`

### Screenshot of functionality 

1. `CommunityIntroDialog` in storybook:

https://user-images.githubusercontent.com/97019400/216627166-e356af83-fc02-4676-8e65-fa6b3df05ded.mov

2. `JoinCommunityView` in storybook:

https://user-images.githubusercontent.com/97019400/216627255-57012b38-d444-4dc6-9a29-4f331b2454d3.mov

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Chat/views/communities/JoinCommunityView.qml [hunk 1]
```diff
@@ -274,4 +284,18 @@ StatusSectionLayout {
...
+
+        name: root.name
+        introMessage: root.introMessage
+        imageSrc: root.image
+        accessType: root.accessType
+        isInvitationPending: root.isInvitationPending
+
+        onJoined: root.joined()
+        onCancelMembershipRequest: root.cancelMembershipRequest()
+
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/communities/JoinCommunityView.qml:299]: whitespace

File: ui/app/AppLayouts/Chat/views/communities/JoinCommunityView.qml [hunk 2]
```diff
@@ -46,6 +50,12 @@ StatusSectionLayout {
     signal infoButtonClicked
     signal adHocChatButtonClicked
     signal revealAddressClicked
+    signal joined
+    signal cancelMembershipRequest
+
+    function  openJoinCommunityDialog() {
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/communities/JoinCommunityView.qml:56]: whitespace

**Review by caybro — APPROVED** [review]
LGTM

**Review by alexjba — APPROVED** [review]
LGTM! 👍

**Review by glitchminer — APPROVED** [review]
Tested, approved.

Summary:
General look and feel checks.
Resizing
Joined community

Will need further testing in app with max / min characters at supported heights/widths.

**Pull request merged**